src/types/options.ts

```diff
@@ -2149,3 +2149,16 @@ export const SCHOOLS: School[] = RAW_SCHOOLS.split('\n')
     state,
     website: website.replace('http://', ''),
   }));
+
+const RAW_TAGS = `athletics
```

Comment:
A few comments:
- Instead of a multiline string just make this a list of strings (should be pretty easy with VS Code multi-cursors). The `RAW` values we have elsewhere are a little more complicated as there are multiple values in one line.
- The casing here is inconsistent `DACA only` but `essay`.
- How do we plan on handling similar tags like `Essay` vs. `essay`?
- I'm curious if for some of these we should just have separate checkboxes in the form that users can check if it's the case. We can store the values as tags under the hood.

Reply:
what tag options could we provide instead then?

Reply:
I feel like this is more of a design question we should discuss with others but for now I feel like we should have either:
- a blank autocomplete without any preset options. E.g. 
> **Keywords**
> [E.g. athletics, daca, essay, stem, etc.]
- how it is right now but with additional info to explain each keyword a little better. Maybe you can make this a map from `'tag'` to `'description'`, override `getOptionLabel` to return the key and `renderOption` to return something like `<strong>${option}</strong>: ${KEYWORDS[options]}` or `<strong>${option}</strong>: ${KEYWORDS[options].description}`. E.g. you might see this in the options:
  - **daca-only**: Only DACA recipients can apply.
  - **stem**: For students majoring or interested in majoring in STEM.
  - **yearly**: This scholarship is offered yearly.

We can lowercase everything so `Daca` and `DACA` and `daca` aren't separate words and replace spaces with `-` so it's more consistent. When users submit a string enter they should see the lowercase value of what they entered with spaces replaced.

Eventually we could maybe additionally add some checkboxes that'll insert some of these keywords. But maybe if the keyword description is clear enough people will know how to add the right ones.

Reply:
went with the first option